ipatests/test_integration/test_installation.py

```diff
@@ -2123,3 +2123,79 @@ def test_hostname_matching_domain(self):
         assert result.returncode == 1
         assert 'hostname cannot be the same as the domain name' \
             in result.stderr_text
+
+
+class TestNsslapdIgnoreTimeSkew(IntegrationTest):
+    """
+    Test to check nsslapd-ignore-time-skew is not disabled.
+    """
+    num_replicas = 1
+    topology = 'line'
+
+    def install(self, mh):
+        tasks.install_master(
+            self.master, setup_dns=True,
+            extra_args=['--no-dnssec-validation']
+        )
+        tasks.install_replica(
+            self.master,
+            self.replicas[0],
+            setup_dns=True,
+            setup_ca=True,
+            extra_args=['--no-dnssec-validation']
+        )
+
+    @pytest.fixture
+    def update_time_skew(self):
+        """
+        Fixture enables nsslapd-ignore-time-skew
+        parameter and reverts it back
+        """
+        ldap = self.replicas[0].ldap_connect()
+        dn = DN(
+            ("cn", "config"),
+        )
+        entry = ldap.get_entry(dn)
+        entry.single_value["nsslapd-ignore-time-skew"] = 'on'
+        ldap.update_entry(entry)
+
+        yield
+
+        entry = ldap.get_entry(dn)
+        entry.single_value["nsslapd-ignore-time-skew"] = 'off'
+        ldap.update_entry(entry)
+
+    def test_check_nsslapd_ignore_time_skew(self):
+        """
+        This testcase check that when replica CA installation
+        is done ignore time skew message is seen during dirsrv
+        request service step. It also checks that the time skew
+        is reverted during pki_tomcat setup stage.
```

Comment:
This testcase checks that the ignore time skew parameter is set to on during the directory server replica installation (replication of the suffix) and during the CA replica (replication of o=ipaca).
It also checks that the time skew is reverted during pki_tomcat setup stage.